draft-ietf-rtcweb-jsep.xml

```diff
@@ -159,7 +159,9 @@ ipr="trust200902">
         its chosen signaling protocol, and perform the reverse
         conversion on the messages it receives from the other side.</t>
 
-        <t>One way to mitigate this is to provide a JavaScript library
+        <t>One way to reduce the complexity of the application's
```

Comment:
This feels wordy. Perhaps "One way to make life easier for the application is to... "